src/Controller/User/ChangeEmailController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Controller\User;
+
+use App\Entity\User;
+use App\Form\ChangeEmailForm;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Attribute\Route;
+use Symfony\Component\Security\Http\Attribute\CurrentUser;
+
+final class ChangeEmailController extends AbstractController
+{
+    #[Route('/change-email', name: 'change_email')]
+    public function changeEmail(
+        Request $request,
+        EntityManagerInterface $em,
+
+        #[CurrentUser]
+        User $user,
+    ) {
+        $form = $this->createForm(ChangeEmailForm::class, $user);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em->flush();
+            $this->addFlash('success', 'You have successfully changed your email.');
+
+            return $this->forward(VerificationController::class.'::sendVerification');
```

Comment:
Do we communicate to the user what's happening?

Reply:
That controller sets a flash message. On the change-email page, we should show a clear message that "Changing your email will require re-verification". I'll add to the form `help` for now but we can re-evaluate during the design phase.